dimod/discrete/cydiscrete_quadratic_model.pyx

```diff
@@ -262,8 +262,8 @@ cdef class cyDiscreteQuadraticModel:
 
     @cython.boundscheck(False)
     @cython.wraparound(False)
-    def add_constraint_as_quadratic(self, object terms,
-                                    Bias lagrange_multiplier, Bias constant):
+    def add_linear_equality_constraint(self, object terms,
```

Comment:
might as well move this up in alphabetical order.

Reply:
Good point, will do